packages/posix-getopt/posix-getopt.3.1.0/opam

```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings for posix getopt/getopt_long"
+description:
+  "posix-getopt provides a simple interface for the POSIX getopt and its extensions, getopt_long and getopt_long_only."
+maintainer: ["romain.beauxis@gmail.com"]
+authors: ["Romain Beauxis"]
+license: "MIT"
+homepage: "https://github.com/savonet/ocaml-posix"
+bug-reports: "https://github.com/savonet/ocaml-posix/issues"
+depends: [
+  "dune" {>= "3.20"}
+  "dune-configurator"
+  "ounit2" {with-test}
+  "process" {with-test}
+  "posix-uname" {with-test & = version}
+  "ctypes"
+  "posix-base" {= version}
+  "unix-errno"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/savonet/ocaml-posix.git"
+x-maintenance-intent: ["(latest)"]
+available: os != "win32" & os != "bsd"
```

Comment:
Here you probably want `os-family != "bsd"` or `os != "freebsd"`
following https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values